sqlglot/dialects/hive.py

```diff
@@ -47,9 +47,31 @@
     "HOUR": " / 3600",
 }
 
+INTERVAL_VARS = {
+    "SECOND",
+    "SECONDS",
+    "MINUTE",
+    "MINUTES",
+    "DAY",
+    "DAYS",
+    "MONTH",
+    "MONTHS",
+    "YEAR",
+    "YEARS",
+}
```

Comment:
Wasn't sure but maybe we should move this in utils?

Reply:
well this will be unique for every dialect cc, @barakalon was taking a look at this

Reply:
Makes sense, will resolve this for now then